src/compiler/steering-meeting.md

```diff
@@ -1,49 +0,0 @@
-# Compiler-team Steering Meeting
```

Comment:
Where did this content go? I will dearly miss this file, I find that it was useful ...

Reply:
I incorporated the details that I thought was useful into the meetings page. If there's a specific detail that is now missing, I can re-add it.